spec/features/create_collection_spec.rb

```diff
@@ -65,4 +65,19 @@
 
     expect(page).not_to have_content('Contributor')
   end
+
+  context "when user is not an admin" do
+    it "Does not display the collections link when the setting is off" do
+      user.remove_role(:admin) # Ensure user is not an admin for this test
```

Comment:
Instead of this, how about setting user in the top describe context to an admin user:
`let(:user) { FactoryBot.create(:admin) }`
then in this context override it:
`let(:user) { FactoryBot.create(:user) }`
I think that would be clearer then how currently the user is an admin just because of the email address which isn't very explicit.
It would also be nice to replicate this context block for `when user is an admin` just to test those cases.